packages/gestalt/src/Callout.js

```diff
@@ -0,0 +1,111 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes, { string } from 'prop-types';
+import Box from './Box.js';
+import Icon from './Icon.js';
+import IconButton from './IconButton.js';
+import Link from './Link.js';
+import Text from './Text.js';
+import typography from './Typography.css';
+
+type Props = {|
+  description: string,
+    iconAccessibilityLabel: string,
+      primaryLink ?: {|
+        href: string,
+          label: string,
+  |},
+onDismiss: () => void,
+  secondaryLink ?: {|
+    href: string,
+      label: string,
+  |},
+style: 'error' | 'info' | 'warning',
+  title ?: string,
```

Comment:
My editor did this whenever I saved. I'll update to make sure it's formatted correctly when I address feedback